ArduPlane/transition.h

```diff
@@ -105,10 +105,10 @@ class SLT_Transition : public Transition
 
 protected:
 
-    enum {
-        TRANSITION_AIRSPEED_WAIT,
-        TRANSITION_TIMER,
-        TRANSITION_DONE
+    enum class State {
```

Comment:
"State" could be used for so many things. How about TransitionState?

Reply:
> "State" could be used for so many things. How about TransitionState?

Within the object it can be refered to as "State", but outside the object it has to be namespaced (eg. `Tailsitter_Transition::State::ANGLE_WAIT_FW`)